### PR TITLE
Refactor output format framework to support tables and empty format 

### PR DESCRIPTION
**Pull request by gowriswarupk**
# Motiviation

Addressing issue #1429 

### Changes made:
---
- fix:Kafka topic creation command "mute" successful. [1]

### Verification Steps:
---
- for fix [1] :
`./rhoas kafka topic create --name=kannan --output=none`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -86,12 +86,12 @@ generate-downstream-docs: ## Generate command-line reference documentation in ad
 
 I18N_LINTER_DEF := $(shell command -v app-services-go-linter 2> /dev/null)
 
-lint-lang: ## Lint i18n files
-ifndef I18N_LINTER_DEF # check if the linter is installed, install it if not
-	go install github.com/redhat-developer/app-services-go-linter/cmd/app-services-go-linter@latest
-endif
-	app-services-go-linter -path ./pkg/core/localize/locales ./...
-.PHONY: lint-lang
+# lint-lang: ## Lint i18n files
```
Comment (wtrocki) [inline Makefile:89]: Remove this

Comment (wtrocki) [inline Makefile:89]: Remove this

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,30 +1,4 @@
 
-<a name="v0.39.0"></a>
```
Comment (wtrocki) [inline CHANGELOG.md:2]: Restore this changelog (generate it)

Comment (wtrocki) [inline CHANGELOG.md:2]: Restore this changelog (generate it)

**Review by wtrocki — APPROVED** [review]
Amazing change. I did not verify this but code looks very good

**Comment by wtrocki** [comment]
We need to resolve conflicts, remove some extra changes. Then myself and @rkpattnaik780 can verify and merge this change

**Comment by rkpattnaik780** [comment]
Looks bit counter-intuitive to see "none" in `list` command. Wdyt?
```
./rhoas kafka topic list -o none
json   none   table  yaml   yml 
```

**Pull request closed without merging**